src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -1369,12 +1369,42 @@ private BoundExpression BindNonMethod(SimpleNameSyntax node, Symbol symbol, Diag
                         }
                         else
                         {
+                            SourceLocalSymbol sourceLocal;
+                            type = null;
+
+                            if (node.SyntaxTree == localSymbolLocation.SourceTree && (object)(sourceLocal = localSymbol as SourceLocalSymbol) != null &&
```

Comment:
Nevermind, I got it :-)

---
In reply to: [64277268](https://github.com/dotnet/roslyn/pull/11493#discussion_r64277268) [](ancestors = 64277268)